install/kubernetes/istio-auth.yaml

```diff
@@ -572,15 +572,20 @@ spec:
   scope: Namespaced
   version: v1alpha2
 ---
-# Pilot service for discovery
+#Istio Configuration file
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: istio-system
+---
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: istio
   namespace: istio-system
 data:
   mesh: |-
-    # Uncomment the following line to enable mutual TLS between proxies
+    # Uncomment the following line to enable mutual encryption
```

Comment:
leave it as mTLS or as "encryption" but not "mutual encryption" that's meaningless

Reply:
this should be left alone as mTLS. we don't support regular TLS which is also encryption.